sass/includes/_media-embed.scss

```diff
@@ -65,12 +65,33 @@
     background-color: $color__dark;
     font-size: 1.2rem;
     margin-top: 2rem;
-    padding: 2rem;
+    padding: 1rem;
+
+    details {
+      summary{
+        text-align: left;
```

Comment:
Hi James - minor point but I wonder if the `text-align: left` would be better placed on the `<details>` element and removed from `&__transcript-text`. Apologies if I'm missing something because I can't actually see the rendered output.

Reply:
Cool, I've made that change